src/PAModel/Entropy.cs

```diff
@@ -71,11 +71,20 @@ internal class PropertyEntropy
 
         public PropertyEntropy VolatileProperties { get; set; }
 
-        // Sometimes, an empty LocalDatabaseReferences is serialized as "" or { }.
-        // Need to track which so we can round-trip.
-        // if true: "".   Else,  "{}". 
+        /// <summary>
```

Comment:
nit : I believe we generally use "//" for single line comments rather than "///" int his repo. Would be great if we could stick to that.